projects/angular/src/forms/common/control-container.ts

```diff
@@ -23,7 +23,6 @@ import { NgControlService } from './providers/ng-control.service';
         <cds-icon
           *ngIf="showInvalid"
           class="clr-validate-icon"
-          status="danger"
```

Comment:
Does this color issue also apply to success below? (this is why I was hoping for a stackblitz, so I could play around with it myself)

Reply:
No, just the danger. You can compare the files:
\clarity-angular\projects\angular\src\forms\common\control-container.ts
\clarity-angular\projects\angular\src\forms\input\input-container.ts

You will see the only difference is the danger status.

Reply:
Btw, use https://github.com/vmware-clarity/ng-clarity/pull/308 instead. I closed this one as I had not yet been added to the vmware group etc. Read somewhere you needed to create a new request.